CODEOWNERS

```diff
@@ -0,0 +1,143 @@
+lib/vdsm/sslutils.py                    @tinez
+lib/vdsm/supervdsm_server.py            @tinez
+lib/vdsm/taskset.py                     @nirs
+lib/vdsm/throttledlog.py                @mz-pdm
+lib/vdsm/tool/                          @tinez
+lib/vdsm/utils.py                       @tinez
+lib/vdsm/v2v.py                         @mz-pdm
+lib/vdsm/vdsm.conf.sample               @tinez
+lib/vdsm/vdsmd.py                       @tinez
+lib/vdsm/virtsparsify.py                @mz-pdm
+lib/vdsm/virtsysprep.py                 @mz-pdm
+
+# Profiling
+lib/vdsm/profiling/                     @nirs
+lib/vdsm/profiling/memory
+
+# Common
+lib/vdsm/common/                        @tinez
+lib/vdsm/common/api.py                  @tinez @nirs
+lib/vdsm/common/base26.py               @mz-pdm
+lib/vdsm/common/cache.py                @nirs @tinez
+lib/vdsm/common/cmdutils.py             @tinez @nirs
+lib/vdsm/common/commands.py             @tinez @nirs
+lib/vdsm/common/compat.py               @tinez @nirs
+lib/vdsm/common/concurrent.py           @nirs
+lib/vdsm/common/config.py.in            @tinez
+lib/vdsm/common/constants.py.in         @tinez
+lib/vdsm/common/conv.py                 @mz-pdm
+lib/vdsm/common/cpuarch.py              @mz-pdm
+lib/vdsm/common/define.py               @mz-pdm
+lib/vdsm/common/dsaversion.py.in        @tinez
+lib/vdsm/common/errors.py               @nirs
+lib/vdsm/common/eventfd.py              @tinez
+lib/vdsm/common/exception.py            @nirs @mz-pdm
+lib/vdsm/common/filecontrol.py          @tinez
+lib/vdsm/common/fileutils.py            @almusil @nirs @mz-pdm
+lib/vdsm/common/function.py             @almusil
+lib/vdsm/common/glob.py                 @nirs @vjuranek
+lib/vdsm/common/hooks.py                @tinez
+lib/vdsm/common/hostdev.py              @mz-pdm
+lib/vdsm/common/hostutils.py            @almusil
+lib/vdsm/common/libvirtconnection.py    @mz-pdm
+lib/vdsm/common/lockfile.py             @tinez @nirs
+lib/vdsm/common/logutils.py             @nirs @mz-pdm @tinez
+lib/vdsm/common/nbdutils.py             @nirs
+lib/vdsm/common/osutils.py              @tinez
+lib/vdsm/common/panic.py                @nirs @tinez
+lib/vdsm/common/password.py             @nirs @mz-pdm @tinez
+lib/vdsm/common/pki.py                  @tinez
+lib/vdsm/common/proc.py                 @almusil
+lib/vdsm/common/properties.py           @nirs @vjuranek
+lib/vdsm/common/pthread.py              @mz-pdm @nirs @tinez
+lib/vdsm/common/response.py             @mz-pdm
+lib/vdsm/common/sigutils.py             @tinez @nirs
+lib/vdsm/common/supervdsm.py            @tinez @nirs
+lib/vdsm/common/systemctl.py            @nirs @vjuranek
+lib/vdsm/common/systemd.py              @nirs @vjuranek
+lib/vdsm/common/threadlocal.py          @tinez @nirs
+lib/vdsm/common/time.py                 @tinez @nirs
+lib/vdsm/common/udevadm.py              @nirs @vjuranek
+lib/vdsm/common/units.py                @nirs @vjuranek
+lib/vdsm/common/unixrpc.py              @mz-pdm
```

Comment:
Infra?

Reply:
```
$ git shortlog -sn lib/vdsm/common/unixrpc.py
     2  Francesco Romani
     1  Andrej Krejcir
     1  Dan Kenigsberg

$ git grep unixrpc
lib/vdsm/momIF.py:from vdsm.common import unixrpc
lib/vdsm/momIF.py:        self._mom = ThrottledClient(unixrpc.UnixXmlRpcClient(
tests/mom_test.py:import unixrpc_testlib
tests/mom_test.py:        server = unixrpc_testlib.UnixXmlRpcServer(self._sock_path)
```

So this looks like mom support, looks like virt to me :-) \

@tinez do you want to own this file?